### PR TITLE
Message deletion confirmation is implemented

### PR DESCRIPTION
**Pull request by nemanja777**
after clicking the delete button, a confirmation dialog opens, after which we can choose to delete the message or exit the dialog

### CODE REVIEW

File: public/chat/santaChat.js [hunk 1]
```diff
@@ -31,11 +34,10 @@ $(async function() {
         $(chatElement).find('[data-name="chatDate"]').text(dateStr);
         $(chatElement).find('[data-name="chatFrom"]').text(`From: ${item.from || 'Anonymous'}`);
 
-        $(chatElement).find('button').on('click', function() {
-          $.post(`${apiUrl}/delete`, { _id: item._id }, result => {
-            if (result.success) $(chatElement).remove();
-            showAlert(result);
-          });
+        $(chatElement).find('[data-name="msgDelete"]').on('click', function() {
+          $('#deletePairDialog').prop('hidden', false);
```
Comment (Zaista) [inline public/chat/santaChat.js:38]: what's this used for?

File: public/chat/santaChat.js [hunk 2]
```diff
@@ -59,13 +61,11 @@ $(async function() {
           $(chatElement).find('[data-name="chatDate"]').text('Just now...');
           $(chatElement).find('[data-name="chatFrom"]').text('From: you');
 
-          $(chatElement).find('button').on('click', function() {
-            $.post(`${apiUrl}/delete`, { _id: response.insertedId }, result => {
-              if (result.success) $(chatElement).remove();
-              showAlert(result);
-            });
+          $(chatElement).find('[data-name="msgDelete"]').on('click', function() {
+            $('#deletePairDialog').prop('hidden', false);
```
Comment (Zaista) [inline public/chat/santaChat.js:65]: here as well, what's this?

File: public/chat/santaChat.js [hunk 3]
```diff
@@ -81,4 +81,13 @@ $(async function() {
   $('.alert .btn-close').on('click', function() {
     $('.alert').hide();
   });
+
+  $('#deleteMessageDialog').on('click', () => {
```
Comment (Zaista) [inline public/chat/santaChat.js:85]: I would prefer this #deleteMessageDialog to be renamed to deleteMessageButton, as it is a button

**Review by Zaista — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -18,6 +18,7 @@
     "cookie-session": "^2.0.0",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
+    "lint": "^0.8.19",
```
[the remark this answers is not in the document]
Comment (nemanja777) [reply package.json:21]: I installed lint, thinking it didn't exist because I didn't write a good command.

File: public/chat/santaChat.js (hunk 1 above)
Comment (nemanja777) [reply public/chat/santaChat.js:38]: With the help of the selector, we find the button with which we open the dialog

File: public/chat/santaChat.js (hunk 2 above)
Comment (nemanja777) [reply public/chat/santaChat.js:65]: I'm not sure, I'd have to think about it

File: public/chat/santaChat.js (hunk 1 above)
Comment (Zaista) [reply public/chat/santaChat.js:38]: I was referring to this line: $('#deletePairDialog').prop('hidden', false);

File: package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (nemanja777) [reply package.json:21]: I deleted it

**Review by Zaista — APPROVED** [review]

**Pull request merged**